ext/Pod-Html/lib/Pod/Html/Util.pm

```diff
@@ -30,75 +30,52 @@ Pod::Html::Util - helper functions for Pod-Html
 
 =head1 SUBROUTINES
 
-=head2 C<parse_command_line()>
+=head2 C<process_command_line()>
 
-TK
+Process command-line switches (options).  Returns a reference to a hash.  Will
+provide usage message if C<--help> switch is present or if parameters are
+invalid.
 
```

Comment:
```suggestion

Calling this subroutine may modify C<@ARGV>.

```

Reply:
Accepted.  We'll review `@ARGV` when we're done to evaluate this and other comments about that array.